packages/calling-component-bindings/src/handlers/createTeamsCallHandlers.ts

```diff
@@ -109,6 +117,33 @@ export const createDefaultTeamsCallingHandlers = memoizeOne(
         }
         /* @conditional-compile-remove(teams-identity-support) */
         await call?.removeParticipant(participant);
+      },
+      /* @conditional-compile-remove(one-to-n-calling) */
+      onAcceptCall: async (incomingCallId: string, useVideo?: boolean): Promise<void> => {
+        const cameras = await deviceManager?.getCameras();
+        let localVideoStream: LocalVideoStream | undefined;
+        if (cameras && useVideo) {
+          localVideoStream = new LocalVideoStream(cameras[0]);
+        }
```

Comment:
Extra line 123 - 127 for all places which needs a LocalVideoStream

from these whole functions onAcceptCall and onRejectCall, we could extra them into 2 isolated functions to align the logic (so we don't need to worry about inconsistency when making more code changes)

Reply:
Due to the Agent typing I am not sure how this might work. Another topic for monday!

Reply:
to create util function for local stream creation